.orchestra/config/components/revng-python-dependencies.yml

```diff
@@ -10,19 +10,19 @@ dependencies:
 - cython
 requirements: |-
   # Python language checkers/formatters
-  black
-  flake8
-  flake8-breakpoint
-  flake8-builtins
-  flake8-comprehensions
-  flake8-eradicate
-  flake8-plugin-utils
-  flake8-polyfill
-  flake8-return
-  flake8-simplify
-  isort
-  mypy
-  pep8-naming
+  black>=22.12.0,<23.0
+  flake8>=5.0.4,<6.0
+  flake8-breakpoint>=1.1.0,<2.0
+  flake8-builtins>=2.1.0,<3.0
+  flake8-comprehensions>=3.10.1,<4.0
+  flake8-eradicate>=1.4.0,<2.0
+  flake8-return>=1.2.0,<2.0
+  flake8-simplify>=0.19.3,<1.0
+  pep8-naming>=0.13.3,<1.0
+  isort>=5.11.4,<6.0
+  mypy==0.982
```

Comment:
Why do you prefer to set ranges instead of exact dependencies? To get minor fixes?

Reply:
Yes, I expect minor releases to not disrupt linting